src/app/features/ledger/generic-steps/connect-device/connect-ledger.tsx

```diff
@@ -0,0 +1,44 @@
+import { useNavigate } from 'react-router-dom';
+
+import { RouteUrls } from '@shared/route-urls';
+
+import { doesBrowserSupportWebUsbApi, whenPageMode } from '@app/common/utils';
+import { openIndexPageInNewTab } from '@app/common/utils/open-in-new-tab';
+import { BaseDrawer } from '@app/components/drawer/base-drawer';
+
+import { ConnectLedgerLayout } from './connect-ledger.layout';
+
+export function ConnectLedger() {
+  const navigate = useNavigate();
+  const pageModeRoutingAction = (url: string) =>
+    whenPageMode({
+      full() {
+        navigate(url);
+      },
+      popup() {
+        void openIndexPageInNewTab(url);
+        window.close();
+      },
+    });
+
+  function connectChain(chain: string) {
+    const supportsWebUsbAction = pageModeRoutingAction(
+      RouteUrls.Onboarding + `/${chain}/` + RouteUrls.ConnectLedger
+    );
+    const doesNotSupportWebUsbAction = pageModeRoutingAction(
+      RouteUrls.Onboarding + '/' + RouteUrls.LedgerUnsupportedBrowser
+    );
+
+    doesBrowserSupportWebUsbApi() ? supportsWebUsbAction() : doesNotSupportWebUsbAction();
+  }
+
+  return (
+    <BaseDrawer isShowing onClose={() => navigate('../')}>
+      <ConnectLedgerLayout
+        connectBtc={() => connectChain('bitcoin')}
+        connectStacks={() => connectChain('stacks')}
```

Comment:
```suggestion
        connectBitcoin={() => connectChain('bitcoin')}
        connectStacks={() => connectChain('stacks')}
```
consistent use of chain name